helm/helm-deploy.sh

```diff
@@ -2,8 +2,6 @@
 
 helm upgrade ${CHART_NAME} \
  ./helm/${CHART_NAME} \
---atomic \
---cleanup-on-fail \
 --install \
 --reset-values \
 --timeout 3m \
```

Comment:
we've seen this is problematic on notprod. I think the value on the rest of the services is 10m